src/js/components/TaskTable.js

```diff
@@ -50,6 +52,15 @@ class TaskTable extends React.Component {
         sortFunction
       },
       {
+        className: this.getHostColumnClassname,
```

Comment:
just curious, is this cacheable? this will matter with large task tables which will be often